gcp/modules/gcp-stackdriver-alerting/resources/notification_channels/alerts_at_raisingthefloor_org.json

```diff
@@ -1,7 +1,7 @@
 {
   "type": "email",
   "labels": {
-    "email_address": "sergey@raisingthefloor.org"
+    "email_address": "alerts+${env}@raisingthefloor.org"
```

Comment:
This is fine, but I think it might be helpful to distinguish between different dev environments (e.g. Alice may want to subscribe to alerts@ and filter out everything but alerts+dev-alice; we all want to filter out alerts+dev-bob because Bob is always breaking stuff in his environment).

How about `var.project_id` or `var.domain_name` instead?

Reply:
I think `alerts+prd.gcp.gpii.net@raisingthefloor.org` is more readable than `alerts+gpii-gcp-prd@raisingthefloor.org`, will update to domain name.